libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -404,6 +404,16 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
             item._noSeperator = this.noSeperator;
             this.stateChanges.next(item);
         });
+
+
+        const indicators = this.itemEl.nativeElement.querySelectorAll('fd-busy-indicator');
+        if (indicators) { 
+            indicators.forEach((indicator) => {
+                if (indicator) {
+                    indicator.setAttribute('aria-label', '');
```

Comment:
Why do we need to reset `aria-label`?

Reply:
to avoid reading "loading" message read by default even tho loading is not initiated. by default busy indicator has this set as loading